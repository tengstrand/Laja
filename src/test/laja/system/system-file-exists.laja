
#write "target/test-output/system-file-exists.out"
  #if (fileExists("{.}/system-file-exists.laja"))
    This template exists!
  #end
  #if (fileExists("{.}/aaa"))
    The file aaa exists!
  #else
    The file aaa does not exists!
  #end
  #if (fileExists("{.}"))
    Current directory is not a file
  #else
    Current directory does not exist, it's a file
  #end
#end
