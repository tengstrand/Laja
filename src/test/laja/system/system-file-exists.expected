    This template exists!
    The file aaa does not exists!
    Current directory does not exist, it's a file
